core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/BoxImpl.java

```diff
@@ -17,12 +28,24 @@
 import uk.ac.bbsrc.tgac.miso.core.data.Boxable;
 import uk.ac.bbsrc.tgac.miso.core.util.BoxUtils;
 
+@Entity
+@Table(name = "Box")
 public class BoxImpl extends AbstractBox implements Serializable {
 
   protected static final Logger log = LoggerFactory.getLogger(BoxImpl.class);
 
   // The contents of the Box
-  private Map<String, Boxable> boxableItems = new HashMap<String, Boxable>();
+  @ManyToAny(metaColumn = @Column(name = "targetType"))
+  @MapKeyColumn(name = "position", unique = true)
+  @MapKeyClass(String.class)
+  @JoinTable(name = "BoxPosition", joinColumns = { @JoinColumn(name = "targetId") }, inverseJoinColumns = {
```

Comment:
There's no column on BoxPosition with the name `targetId`, did you mean `boxPositionId`? (this affects the `@Formula` annotations too)